package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "inventory-management-api",
-  "version": "0.0.9",
+  "version": "0.0.10",
```

Comment:
Please don't forget to increase the version for each task. @RamaMahjoub

Reply:
👍🏻